changebot/github/github_api.py

```diff
@@ -133,7 +132,7 @@ def get_user_config(self, path_to_file='pyproject.toml',
         try:
             file_content = self.get_file_contents(path_to_file)
             cfg = toml.loads(file_content)
-            cfg = cfg['tool'][current_app.bot_username]
```

Comment:
Sorry, @Cadair , I have to undo this one. Customization like this ideally should be part of a larger effort as #2 or #58.